src/customnodesession.cpp

```diff
@@ -179,10 +189,9 @@ Status CustomNodeSession::createBlob(const struct CustomNodeTensor* tensor, Infe
         return StatusCode::NODE_LIBRARY_INVALID_SHAPE;
     }
     InferenceEngine::SizeVector shape(tensor->dims, tensor->dims + tensor->dimsCount);
-    desc.setDims(shape);
 
     size_t expectedElementsCount = std::accumulate(std::begin(shape), std::end(shape), 1, std::multiplies<size_t>());
-    size_t expectedDataLength = expectedElementsCount *= ovmsPrecisionToIE1Precision(precision).size();
+    size_t expectedDataLength = expectedElementsCount *= ov::element::Type(precision).size();  // TODO
```

Comment:
TODO what?

Reply:
Already done - newest change was not pushed to remote.